src/main/java/org/jenkinsci/plugins/googleplayandroidpublisher/ApkPublisher.java

```diff
@@ -260,6 +271,22 @@ private double getExpandedRolloutPercentage() throws IOException, InterruptedExc
         return expanded;
     }
 
+    private String getExpandedInAppUpdatePriorityString() throws IOException, InterruptedException {
+        return expand(getInAppUpdatePriority());
+    }
+
+    private Integer getExpandedInAppUpdatePriority() throws IOException, InterruptedException {
+        try {
+            String value = getExpandedInAppUpdatePriorityString();
+            if (value != null) {
+                return Integer.parseInt(value.trim());
+            }
+            return null;
+        } catch (NumberFormatException e) {
+            return null;
```

Comment:
Would be nice to print the error so that the console will show why it resolved to null. I.e. if they entered an undefined variable, it won't be parsable as an integer, and we should show that feedback in the build console

Reply:
I'm far from an expert on Jenkins plugin but I don't have any logger here, how should I do ? In fact it should be catch before in isConfigValid

Reply:
You're right, the logger would handle this in isConfigValid() -- at least, now that it is checking the expanded value and not the original string.